homeassistant/components/light/demo.py

```diff
@@ -129,9 +137,14 @@ def turn_on(self, **kwargs):
         if ATTR_EFFECT in kwargs:
             self._effect = kwargs[ATTR_EFFECT]
 
+        # For non-polling lights it is necessary to schedule an update
+        # after making a change, here just to give an example.
```

Comment:
Will change that. I was under impression that it is not really needed in turn_on & turn_off, as these are called by the homeassistant.